radixdlt/src/main/java/com/radixdlt/consensus/PendingVotes.java

```diff
@@ -43,21 +46,27 @@
 	public Optional<QuorumCertificate> insertVote(Vote vote, ValidatorSet validatorSet) {
 		Objects.requireNonNull(vote, "vote");
 
-		Hash voteId = vote.getVertexMetadata().getId();
+		Hash voteHash;
+		try {
+			voteHash = Hash.of(DefaultSerialization.getInstance().toDson(vote.getVoteData(), Output.HASH));

```

Comment:
I would rather inject a hasher here than have a static dependency on DefaultSerialization (which is hard to test)